examples/opensora_pku/README.md

```diff
@@ -2,12 +2,13 @@
 
 Here we provide an efficient MindSpore version of [Open-Sora-Plan](https://github.com/PKU-YuanGroup/Open-Sora-Plan/tree/main) from Peking University. We would like to express our gratitude to their contributions! :+1:
 
-**OpenSora-PKU is still under active development.** Currently, we are in line with **Open-Sora-Plan v1.1.0**.
+**OpenSora-PKU is still under active development.** Currently, we are in line with **Open-Sora-Plan v1.2.0**.
```

Comment:
add a link to v1.1 branch v0.2.0

Reply:
I submitted a new PR to maintain opensora-pku v1.1.0 in branch v0.2.0. Waiting to be merged.

Reply:
merged